unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as functions from './numbers';
```

Comment:
Why not just import each individual function?

Reply:
Personal preference for brevity, but I see how one might want to list the names of the functions one imported for reference.